plugins/commandParser.js

```diff
@@ -0,0 +1,223 @@
+import { Chat } from '@/store/chat';
+import { VStore } from "@/store";
+
+let _store, _ga, _sentry,
+    _store_state, _store_commit;
```

Comment:
`_store_state` - shorthand for Chat store state
`_store_commit` - shorthand for Chat store commit